pkg/scheduler/controller/shoot/scheduler_control.go

```diff
@@ -151,12 +151,11 @@ func (c *defaultControl) ScheduleShoot(ctx context.Context, obj *gardencorev1bet
 		if _, ok := err.(*common.AlreadyScheduledError); ok {
 			return nil
 		}
-		c.reportFailedScheduling(shoot, err)
+		schedulerLogger.Errorf("Failed Scheduling Shoot %+v onto Seed %s: %v", shoot, seed.Name, err)
```

Comment:
If you use `%+v` with `shoot` this will probably be a very lengthy log message, right?
```suggestion
		schedulerLogger.Errorf("Failed Scheduling Shoot %q onto Seed %q: %v", shoot.Namespace+"/"+shoot.Name, seed.Name, err)
```